### ISSUE TITLE
Can't send character sequences like "^2"

### ISSUE THREAD

**Issue by dgw** [ircanywhere/ircanywhere#258]
When attempting to send a message like `>11^2`, the message actually sent is `>11`. `^^2` => `^`.

**Comment by rickihastings** [comment]
There is a reason for this `^2something` will transform it to mIRC colours. So something will be blue.

![image](https://cloud.githubusercontent.com/assets/45660/5982073/848d5156-a8b7-11e4-9b8c-9b39ca6194c9.png)

It seems like it can't be escaped properly though `\^2` should display `^2` and no colours.

**Comment by chauffer** [comment]
Is the project going anywhere, considering there're **still** issues with sending messages?

**Comment by spencerthayer** [comment]
@chauffer +1

**Comment by MarkCole** [comment]
Yes there are *still* issues with some edge cases such as this.  And the timeline for fixing them is, as it has always been *eventually*. 

There is no one actively working on this issue though, so feel free to submit a PR with a fix @chauffer @spencerthayer. Alternatively there are a number of good IRCAnywhere alternatives should you need something more stable with more active development.
